front/components/assistant/AssistantPicker.tsx

```diff
@@ -44,6 +44,14 @@ export function AssistantPicker({
     setSearchedAssistants(filterAndSortAgents(assistants, searchText));
   }, [searchText, assistants]);
 
+  const searchbarRef = (element: HTMLInputElement) => {
```

Comment:
If the delay is here to ensure the component is mounted, there is a more elegant solution that consist of using `useEffect` with an empty array to ensure it's called only once after the component has been mounted.

Reply:
The delay is mainly here to make sure the inputbar doesn't get the focus

Reply:
We tried your `useEffect` approach but didn't work as the input bar was getting the focus

Reply:
👀

Reply:
Then I guess the issue is that we have some race condition with a code that triggers the focus on the input bar, no?

Reply:
Basically you are fighting this guy: https://github.com/dust-tt/dust/blob/3b80285dbbd7df463df30a6922601f05e1c0ef9e/front/components/assistant/conversation/input_bar/InputBarContainer.tsx#L80-L84

Reply:
To overcome this I guess we would need to set focus on the inputbar if we click anywhere but the 3 action buttons? 🥵